### ISSUE TITLE
Nightly 6.5 5-3-Enhanced-Linked-Mode failure: govc cluster.add

### ISSUE THREAD

**Issue by anchal-agrawal** [vmware/vic#5261]
From `output.xml` (attached):

Test `5-3-Enhanced-Linked-Mode`

```
<arg>Add ESX host to the VC</arg>
</arguments>
<status status="PASS" endtime="20170525 02:09:57.967" starttime="20170525 02:09:57.965"></status>
</kw>
<kw type="for" name="${ip} IN [ ${esx1-ip} | ${esx2-ip} | ${esx3-ip} ]">
<kw type="foritem" name="${ip} = 10.160.23.98">
<kw name="Log To Console" library="BuiltIn">
<doc>Logs the given message to the console.</doc>
<arguments>
<arg>Adding ${ip} to VC</arg>
</arguments>
<status status="PASS" endtime="20170525 02:09:57.970" starttime="20170525 02:09:57.968"></status>
</kw>
<kw name="Run" library="OperatingSystem">
<doc>Runs the given command in the system and returns the output.</doc>
<arguments>
<arg>govc cluster.add -hostname=${ip} -username=root -dc=ha-datacenter -password=e2eFunctionalTest -noverify=true</arg>
</arguments>
<assign>
<var>${out}</var>
</assign>
<msg timestamp="20170525 02:09:57.978" level="INFO">Running command 'govc cluster.add -hostname=10.160.23.98 -username=root -dc=ha-datacenter -password=e2eFunctionalTest -noverify=true 2&gt;&amp;1'.</msg>
<msg timestamp="20170525 02:10:00.058" level="INFO">${out} = 
[K[25-05-17 02:09:58] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... 
[K[25-05-17 02:09:58] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (0%)
[K[25-05-17 02:09:58] adding 1...</msg>
<status status="PASS" endtime="20170525 02:10:00.058" starttime="20170525 02:09:57.971"></status>
</kw>
<kw name="Should Contain" library="BuiltIn">
<doc>Fails if ``container`` does not contain ``item`` one or more times.</doc>
<arguments>
<arg>${out}</arg>
<arg>OK</arg>
</arguments>
<msg timestamp="20170525 02:10:00.060" level="FAIL">'
[K[25-05-17 02:09:58] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... 
[K[25-05-17 02:09:58] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (0%)
[K[25-05-17 02:09:58] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (0%)
[K[25-05-17 02:09:59] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (0%)
[K[25-05-17 02:09:59] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (0%)
[K[25-05-17 02:09:59] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (80%)
[K[25-05-17 02:09:59] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (80%)
[K[25-05-17 02:09:59] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (80%)
[K[25-05-17 02:09:59] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (80%)
[K[25-05-17 02:09:59] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (80%)
[K[25-05-17 02:09:59] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (80%)
[K[25-05-17 02:09:59] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (80%)
[K[25-05-17 02:09:59] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... (80%)
[K[25-05-17 02:10:00] adding 10.160.23.98 to cluster /ha-datacenter/host/cls... Error: A general system error occurred: Unable to get signed certificate forhost name '10.160.23.98' ip '10.160.23.98': Error: Operation failed with error = ERROR_ACCESS_DENIED (5) (5)
govc: A general system error occurred: Unable to get signed certificate forhost name '10.160.23.98' ip '10.160.23.98': Error: Operation failed with error = ERROR_ACCESS_DENIED (5) (5)' does not contain 'OK'</msg>
```

[5-3-Enhanced-Linked-Mode.zip](https://github.com/vmware/vic/files/1030646/5-3-Enhanced-Linked-Mode.zip)

**Comment by anchal-agrawal** [comment]
#5199 has a failure in the same step, albeit with a different error.

**Comment by chengwang86** [comment]
I assume we might see this error again in the future since #5987 does not really target at fixing this issue. Here is the comment from @dougm 
```
That error means VC couldn't get the certificate at all. ... But here is a similar example 
https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=2123386
VC logs may have more info
```

**Comment by chengwang86** [comment]
Seen again in the latest nightly test 5-3-ELM on vsphere 6.5

From log.html
```
[K[04-09-17 01:28:44] adding 10.160.198.179 to cluster /ha-datacenter/host/cls... Error: A general system error occurred: Unable to get signed certificate forhost name '10.160.198.179' ip '10.160.198.179': Error: Operation failed with error = ERROR_ACCESS_DENIED (5) (5)
govc: A general system error occurred: Unable to get signed certificate forhost name '10.160.198.179' ip '10.160.198.179': Error: Operation failed with error = ERROR_ACCESS_DENIED (5) (5)' does not contain 'OK'
```

**Comment by jzt** [comment]
[5-3-Enhanced-Linked-Mode.zip](https://github.com/vmware/vic/files/1350452/5-3-Enhanced-Linked-Mode.zip)

**Comment by mhagen-vmware** [comment]
fixed with #6530
